src/NextGen/DonationForm/resources/registrars/templates/fields/Amount.tsx

```diff
@@ -34,23 +35,22 @@ export default function Amount({
                 >
                     <Label />
                 </label>
-                <div className={classNames('givewp-fields-amount__input--container', {invalid: fieldError})}>
+                { !!allowCustomAmount && (<div className={classNames('givewp-fields-amount__input--container', {invalid: fieldError})}>
```

Comment:
@jonwaldstein I have this conditionally rendering the custom amount field. Is there a reason why it should instead just be visually hidden? I seem to recall the presence of the input being important, but I do not recall why.

Reply:
@kjohnson yes the input holds the field name and value for the form to consume - so it always needs to be present (even if just visually hidden).

Reply:
I reverted this render condition. We can review the custom amount implementation at a later time.